substratevm/mx.substratevm/suite.py

```diff
@@ -1549,19 +1549,10 @@
             "layout" : {
                 "clibraries/" : ["extracted-dependency:substratevm:SVM_HOSTED_NATIVE"],
                 "builder/clibraries/" : ["extracted-dependency:substratevm:SVM_HOSTED_NATIVE"],
+                "builder/lib/" : ["dependency:com.oracle.svm.native.reporterchelper"],
```

Comment:
@fniephaus this still leaves `reporterchelper` under a non-platform-dependent folder:

```
$ find -name "*chelper*"                                                                                        
./lib/svm/builder/clibraries/linux-amd64/liblibchelper.a
./lib/svm/builder/lib/libreporterchelper.so
./lib/svm/clibraries/linux-amd64/liblibchelper.a
```

Reply:
Yes, that's right. We'd only add platform-specific folders for consistency. It would, however, make the consuming Java code unnecessarily complex. That's why I think we can drop the platform-specific folders here. Compared with the other libs, `reporterchelper` is only needed for the image generation runner, not for the native image binary (hence the distinction between "clibraries" and "lib").

Reply:
OK, thanks for the clarification Fabio!

Reply:
Thanks for double-checking! :)